packages/lodestar-spec-test-util/src/downloadTests.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import tar from "tar";
+import stream from "stream";
+import {promisify} from "util";
+
+const BASE_URL = "https://github.com/ethereum/eth2.0-spec-tests/releases/download";
+
+// eslint-disable-next-line prettier/prettier
+const TESTS = ["general", "mainnet", "minimal"];
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+export async function downloadTests(
+  {specVersion, outputDir}: {specVersion: string; outputDir: string},
+  log?: (msg: string) => void
+): Promise<void> {
+  if (log) log(`outputDir = ${outputDir}`);
+
+  await Promise.all(
+    TESTS.map(async (test) => {
+      const URL = `${BASE_URL}/${specVersion}/${test}.tar.gz`;
+      const OUTPUT = `${outputDir}`;
```

Comment:
I think we should separate them by version:
```suggestion
      const OUTPUT = `${outputDir}/${specVersion}`;
```

Reply:
We should review how to deal with version changes. The tests are really heavy so maybe we should keep only a single version locally. What do you think?

Reply:
Yeah we can remove all non {specVersion} directories. Maybe we should have flag for cleanup? I would much rather keep spec tests locally instead of downloading everytime if I'm switching between versions.

Reply:
Sounds good. Should I bring `yargs` at this point, or it's overkill?

Reply:
probably overkill